src/pages/CypressTrigger/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect } from 'react';
+import { render } from 'react-dom';
+
+import Icon from '../Common/Icon';
+import { usePreferredLibrary, useRecordingState } from '../Common/hooks';
+import { endRecording } from '../Common/endRecording';
+import { ScriptType } from '../types';
+
+import FaStyle from '@fortawesome/fontawesome-svg-core/styles.css';
+import CommonStyle from '../Common/styles.css';
+
+function TriggerButton() {
+  // Always set library to Cypress
+  const [_, setPreferredLibrary] = usePreferredLibrary();
+  useEffect(() => {
+    setPreferredLibrary(ScriptType.Cypress);
+  }, []);
+
+  const [recordingTabId] = useRecordingState();
+
+  const isRecording = recordingTabId != null;
```

Comment:
nit: same idea here, if tabeId is not going to be undefined. we can do `!== null` here